core/variant/dictionary.cpp

```diff
@@ -195,6 +195,15 @@ bool Dictionary::has_all(const Array &p_keys) const {
 	return true;
 }
 
+Variant Dictionary::find(const Variant &p_value) const {
```

Comment:
I think this should be called `find_key` or `find_by_value` as clarification since all methods in Dictionary operate by key.